test/integration/mockOauth2Client.ts

```diff
@@ -34,6 +34,17 @@ class MockOAuth2Client implements IOAuth2Client {
   }
   async exchangeCodeForTokenResponse(code: string) {
     if (this.params.client_id === "otherSocialClientId") {
+      const otherClientAccesToken = await createTokenExample({
+        // copied from Facebook id_token
```

Comment:
```suggestion
        // copied from Facebook access_token
```
:question: 

Would seem odd to copy them from the Facebook ID token. I assumed the access tokens of Facebook & Google were opaque

Reply:
Hmm.. that's a good point. Didn't consider that. So we need to get the sub from the id_token